test/test.sh

```diff
@@ -146,6 +150,14 @@ git_good reset case5.cpp
 git_good add case6.cpp
 test_commit_bad "Add: NL at EOF"
 git_good reset case6.cpp
+git_good rm case61.cpp
+test_commit_good "Remove: File with missing NL at end"
+sed -i '{s/bar/pub/}' case62.cpp
```

Comment:
`sed -i` is technically non-standard, but the alternatives are far more verbose.